sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2631,6 +2631,35 @@ pi_int32 ExecCGCommand::enqueueImpCommandBuffer() {
     MEvent->setSyncPoint(OutSyncPoint);
     return PI_SUCCESS;
   }
+  case CG::CGTYPE::CopyAccToPtr: {
+    CGCopy *Copy = (CGCopy *)MCommandGroup.get();
+    Requirement *Req = (Requirement *)Copy->getSrc();
+    AllocaCommandBase *AllocaCmd = getAllocaForReq(Req);
+
+    MemoryManager::ext_oneapi_copyD2H_cmd_buffer(
+        MQueue->getContextImplPtr(), MCommandBuffer, AllocaCmd->getSYCLMemObj(),
+        AllocaCmd->getMemAllocation(), Req->MDims, Req->MMemoryRange,
+        Req->MAccessRange, Req->MOffset, Req->MElemSize, Copy->getDst(),
+        Req->MDims, Req->MAccessRange,
+        /*DstOffset=*/{0, 0, 0}, Req->MElemSize, std::move(MSyncPointDeps),
```

Comment:
To be consistent with the `/*SrcOffset*/ {0, 0, 0} ` comment, would format as this (or whatever clang-format says)
```suggestion
        /*DstOffset*/ {0, 0, 0}, Req->MElemSize, std::move(MSyncPointDeps),
```